src/lib.rs

```diff
@@ -133,3 +133,11 @@ fn compile_impl(resource_file: &Path) {
         println!("cargo:rustc-link-lib=dylib={}", prefix);
     }
 }
+
+/// Find build tools other than the compiler and linker.
+///
+/// This will try its hardest to find tools such as `MIDL.EXE` in Windows Kits and/or SDK directories.
+/// The compilers and linkers can be better found with the `cc` or `vswhom` crates.
```

Comment:
This *needs*:
  * the input format,
  * an example – you seem to be doing something with `MIDL.EXE`, so a simple use-case for that would be nice,
  * a note for non-MSVC targets.

Reply:
I'm not clear about what you mean by "input format". I have tried to hit the other two points in 57685cd.

Reply:
The input format is `"midl.exe"` (an alternative read would've been "midl", or "linker"), but you've included this in the example.